src/AgDatabase.cs

```diff
@@ -156,7 +153,7 @@ public void DropLogin(LoginProperties login)
 
     public void DropAllLogins()
     {
-      foreach(var loginProp in AssociatedLogins()) DropLogin(loginProp);
```

Comment:
Originally we were dropping logins by reading the logins on the primary database which should be accessible while the others aren't. At this point, with the databases being removed from the AG are databases on all nodes readable?

Reply:
Well, the idea here is to drop the logins before the database, so they are all readable.
What you mentioned, `reading the logins on the primary database which should be accessible while the others aren't` is true.  But the logic in the code was only deleting those logins on the Primary node, not all nodes.

The other case we have in our application is... we can delete the DB, then do a 
agDb.DropLogin("<login_name_here>");
agDb.DropLogin("<login_name_here>_reader");

which does not require reading secondary database properties.  Its a no-throw attempt at removing Server level logins only